cmake/winml_cppwinrt.cmake

```diff
@@ -0,0 +1,220 @@
+# This script adds cppwinrt support for VS-generated projects.
```

Comment:
Please add license header and copyright notice.

Reply:
Added